test/storage.test.js

```diff
@@ -56,6 +63,7 @@ describe('Storage tests for feeds', () => {
 
 describe('Storage tests for posts', () => {
   const testPost = {
+    user: 'foo',
```

Comment:
Remove `user` from all the `Post` objects in here.

Reply:
All removed!